tests/stubs/App/Http/Controllers/PeopleControllerStub.php

```diff
@@ -0,0 +1,41 @@
+<?php declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+
+final class PeopleControllerStub
+{
+    public function __invoke(Request $request)
+    {
+        return new JsonResponse([
+            'first_name' => $request->input('first_name'),
+            'last_name' => $request->string('last_name'),
+            'age' => $request->integer('age'),
+            'has_children' => $request->boolean('has_children'),
+            'children' => [
+                'name' => $request->only(['name']),
+            ],
+            'email' => $request->safe()->email, // @phpstan-ignore method.notFound
```

Comment:
While the `safe()` method does not exist on the Illuminate Request class, it is allowed. So ignoring the `method.notFound` error, in favour of being able to assert against it.